wdae/wdae/users_api/models.py

```diff
@@ -226,14 +266,37 @@ def get_last_login_for(email: str):
     @staticmethod
     def is_user_locked_out(email: str):
         last_login = AuthenticationLog.get_last_login_for(email)
-        return (last_login is not None
-                and last_login.failed_attempt > LOCKOUT_THRESHOLD)
+        return (
+            last_login is not None
+            and last_login.failed_attempt > LOCKOUT_THRESHOLD
+            and AuthenticationLog.get_remaining_lockout_time(email) > 0
+        )
+
+    @staticmethod
+    def get_locked_out_error(email: str):
+        seconds_left = AuthenticationLog.get_remaining_lockout_time(email)
+        print(seconds_left)
```

Comment:
This probably should be a debug log.

Reply:
Removed, was used to debug some strange behavior that is fixed.